libs/ledger/include/ledger/chain/block.hpp

```diff
@@ -69,6 +70,7 @@ class Block
   uint64_t       timestamp{0u};      ///< The number of seconds elapsed since the Unix epoch
   BlockEntropy   block_entropy;      ///< Entropy that determines miner priority for the next block
   Weight         weight = 1;         ///< Block weight
+  uint64_t       chain_label{0};     ///< The label of a heaviest chain this block once belonged to
```

Comment:
It is not entirely clear to me what the chain label could mean in this context?

Reply:
It's a number that blocks on the heaviest chain share, from tip down to some number, purportedly the first fork.

Reply:
Commented in block.hpp and main_chain.hpp.